src/coreclr/vm/amd64/redirectedhandledjitcase.S

```diff
@@ -16,10 +16,10 @@
 
         //
         // IN: rdi: original IP before redirect
-        //     rsi: Rip from the Thread::GetAbortContext()
         //
 
         mov             rdx, rsp
+        mov             rsi, rdi
```

Comment:
Where is the `rsi` value used?

Reply:
In the RedirectForThrowControl2 that this macro gets as the `target` argument and calls it.